io/plugins/eu.esdihumboldt.hale.io.gml/src/eu/esdihumboldt/hale/io/gml/reader/internal/StreamGmlReader.java

```diff
@@ -58,6 +61,13 @@
 	 */
 	public static final String PARAM_IGNORE_NAMESPACES = "ignoreNamespaces";
 
+	/**
+	 * The name of the parameter specifying the maximum number of features to
+	 * retrieve per single WFS GetFeature request. Only useful if source is a
+	 * WFS GetFeature request URI.
+	 */
+	public static final String PARAM_FEATURES_PER_WFS_REQUEST = "featuresPerWfsRequest";
```

Comment:
The parameter needs also to be listed in the respective provider extensions that register readers based on StreamGmlReader.